### DISCUSSION TITLE
PPStructure中的表格识别能否以文字检测和识别的结果作为输入

### DISCUSSION THREAD

**Discussion by XiaokangLei** [PaddlePaddle/PaddleOCR#12334] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：Windows 11
- 版本号/Version：ppocr v4 Paddle：2.4.2  PaddleOCR：v4 问题相关组件/Related components：
- 运行指令/Command Code：
- 完整报错/Complete Error Message：

请尽量不要包含图片在问题中/Please try to not include the image in the issue.

PPStructure中的表格识别能否以文字检测和识别的结果作为输入。

**Comment by tink2123** [comment]
表格识别结果就是和文本检测+文本识别结果结合起来的

**Comment by XiaokangLei** [comment]
> 表格识别结果就是和文本检测+文本识别结果结合起来的

接口那块可以指定检测识别模型，但是没有直接将检测和识别结果的list作为输入的。因为有时候我会将检测识别结果做调整，比如文本方向小角度的调整
